crates/runtime/src/datafusion/builder.rs

```diff
@@ -257,6 +260,14 @@ pub fn get_analyzer_rules() -> Vec<Arc<dyn AnalyzerRule + Send + Sync>> {
     ]
 }
 
+/// Custom Spice physical optimizer rules
+#[must_use]
+pub fn get_physical_optimizer_rules() -> Vec<Arc<dyn PhysicalOptimizerRule + Send + Sync>> {
+    let mut rules = PhysicalOptimizer::new().rules;
```

Comment:
Unless you explicitly do this, calling `with_physical_optimizer_rules` isn't "with defaults + ours".